mlrun/datastore/azure_blob.py

```diff
@@ -71,6 +93,82 @@ def get_storage_options(self):
         )
         return self._sanitize_storage_options(res)
 
+    def do_connect(self, storage_options):
+        """Connect to the BlobServiceClient, using user-specified connection details.
+        Tries connection string first, then credential, then account key, SAS token, and finally anonymous login.
+
+        Raises
+        ------
+        ValueError if none of the connection details are available
+        """
+        # based on do_connect in AzureBlobFileSystem.
```

Comment:
```suggestion
        # based on do_connect in AzureBlobFileSystem
```
Please include a hyperlink.